README.md

```diff
@@ -38,6 +38,9 @@ The macro will be called with the argument `VALUE=`_fields-value_.
 If the active spool is cleared in Moonraker, this agent will call (if available):
 `_SPOOLMAN_CLEAR_FIELDS`
 
+The whole process will be called in the terminal (if available): 
```

Comment:
This doesn't make sense to me.

Move it before the paragraph above and write something like:

"After all the macros have been called, `_SPOOLMAN_DONE` will be called, if available."